### ISSUE TITLE
[Android] Fix PTT button for screen reader (TalkBack) users

### ISSUE THREAD

**Issue by VanceVagell** [VanceVagell/kv4p-ht#12]
In my own testing, the only major TalkBack issue is with the PTT button. Because I've implemented it as a momentary button, the screen reader can't successfully hold it down. Need to figure out the right way to do this (maybe we can detect TalkBack, and switch to "Sticky PTT" mode?)

**Comment by VanceVagell** [comment]
With some input from a community member who uses TalkBack, you can actually trigger a momentary button by doing a double tap where the 2nd tap is held. When doing this, the PTT button actually works as-is and does not have any issues.

However, in "Sticky PTT" mode, you would expect a quick double-tap to toggle it on, and another quick double-tap to turn it off. But it still requires double tap and hold like in the non-sticky mode.

So this bug is really that "sticky PTT" doesn't work with the expected double-tap gesture in TalkBack.

**Comment by VanceVagell** [comment]
This is fixed in beta version 12.

The fix was to add an setOnClickListener to the PTT button, which only handles the event if the touch handler did not. This can only happen for TalkBack users (typically users who tap the button will always get the touch handlers, which will consume the event so it never reaches the click handler).
